test/watch/index.js

```diff
@@ -263,6 +263,91 @@ describe('rollup.watch', () => {
 				});
 		});
 
+		it('watches and rebuilds transform dependencies', () => {
```

Comment:
Nice test! I would love to see a few more edge cases handled in this or a second test:
* It works correctly if a dependency is both a real and a manually declared dependency
* I also works if we add a dependency on the second run but not on the first
* what happens if a dependency is missing or deleted? If it is still required, I guess we should throw an error.
* if after a change, the hook no longer requires a dependency and this dependency is deleted at the same time, no error should occur

Reply:
Done :)